functions-app/report-psychometrician/index.js

```diff
@@ -2,15 +2,15 @@
 
 const { performance } = require('perf_hooks')
 
-const v1 = require('./v1')
+const v2 = require('./v2')
 const name = 'report-psychometrician'
 
-module.exports = async function (context, message) {
+module.exports = async function (context, blob) {
   const start = performance.now()
 
   let meta
   try {
-    meta = await v1.process(context.log)
+    meta = await v2.process(context.log, blob)
```

Comment:
I know implementation versions is a well spread pattern that we kept for good reasons but in that particular case do we really need to keep v1? 🤔 
If you wish to keep it for historical reasons you can always find it through github.

Reply:
Yes, for now.